pkg/hintrunner/zero/zerohint_uint256.go

```diff
@@ -190,3 +190,36 @@ func createSplit64Hinter(resolver hintReferenceResolver) (hinter.Hinter, error)
 
 	return newSplit64Hint(a, low, high), nil
 }
+
+func newUint256SignedNN(a hinter.ResOperander) hinter.Hinter {
+	return &GenericZeroHinter{
+		Name: "Uint256SignedNN",
+		Op: func(vm *VM.VirtualMachine, _ *hinter.HintRunnerContext) error {
+			// memory[ap] = 1 if 0 <= (ids.a.high % PRIME) < 2 ** 127 else 0
+			apAddr := vm.Context.AddressAp()
+
+			_, aHigh, err := GetUint256AsFelts(vm, a)
+			if err != nil {
+				return err
+			}
+			var v memory.MemoryValue
+			felt127 := new(fp.Element).SetBigInt(new(big.Int).Lsh(big.NewInt(1), 127))
```

Comment:
Maybe we could add this to `utils/constant.go` file, as it is used in other places as well

Reply:
I think it's a good idea, I have seen it in some tests. I will add felt127 const in this PR, but maybe a new PR with replacements.